eventsourcing/src/main/java/org/axonframework/eventsourcing/AsyncEventSourcingRepository.java

```diff
@@ -97,16 +97,14 @@ public ManagedEntity<ID, M> attach(@Nonnull ManagedEntity<ID, M> entity,
 
     @Override
     public CompletableFuture<ManagedEntity<ID, M>> load(@Nonnull ID identifier,
-                                                        @Nonnull ProcessingContext processingContext,
-                                                        long start,
-                                                        long end) {
+                                                        @Nonnull ProcessingContext processingContext) {
         var managedEntities = processingContext.computeResourceIfAbsent(managedEntitiesKey, ConcurrentHashMap::new);
 
         return managedEntities.computeIfAbsent(
                 identifier,
                 id -> eventStore.transaction(processingContext, context)
                                 .source(
-                                        SourcingCondition.conditionFor(start, end, criteriaResolver.resolve(id)),
```

Comment:
How we will support reading a part of the stream? What's the plan @abuijze?
Because I see just deletion of that possibility, but nothing as replacement.

Reply:
The method on the repository shouldn't declare a start and end, because it assumes event sourcing. If for any other usecase we need some methods back, we can add them as part of implementing that use-case.
But the method as it was didn't make sense. You don't start streaming at a random position in the stream. If you use snapshots, you would use that as the initial state and apply events since the snapshot to that state. Even for that use-case, the previous method signature didn't make sense.